src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -45,6 +45,9 @@
   <ItemGroup Label="Package references">
     <PackageReference Include="Microsoft.VisualStudio.ComponentModelHost" Version="14.0.25424" />
     <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="14.0.23205" />
+    <PackageReference Include="Microsoft.VisualStudio.Imaging" Version="15.0.26201" />
+    <PackageReference Include="Microsoft.VisualStudio.ImageCatalog" Version="14.3.25407" />
+    <PackageReference Include="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" Version="16.7.30328.74" />
```

Comment:
I see we're already referencing this in IssueViz, but I don't understand why it's version 16.

Reply:
I'll see what happens if I change it.